src/platform/packages/private/kbn-grid-layout/grid/grid_row/grid_row.tsx

```diff
@@ -103,46 +102,43 @@ export const GridRow = React.memo(
          */
         const gridLayoutSubscription = gridLayoutStateManager.gridLayout$.subscribe(
           (gridLayout) => {
+            if (!gridLayout[rowIndex]) return;
```

Comment:
This prevents a race condition that is caused by `gridLayoutStateManager.gridLayout$` emitting, which triggers this subscription, when the panel is deleted but **before** this component is unmounted.